src/iframe-helper.js

```diff
@@ -416,8 +416,12 @@ export class SubscriptionApi {
    * @param {boolean} is3p set to true if the iframe is 3p.
    * @param {function(!JsonObject, !Window, string)} requestCallback Callback
    *     invoked whenever a new window subscribes.
```

Comment:
(However, when the remote frame does come from the same origin as the publisher page, those exceptions are not thrown. But we don't have to worry about that, since it will never happen in the real world...right?)